app/models/standard.rb

```diff
@@ -0,0 +1,4 @@
+class Standard < ApplicationRecord
+    has_many :subjects 
+    has_many :users
```

Comment:
@rubyboy-hub  This is incorrect as per the ER diagram

Reply:
Applied new er diagram